CSharpLearning/Set1.Task6/Program.cs

```diff
@@ -0,0 +1,89 @@
+﻿//Add decision logic to your code using the if-elseif -else statement
+//5 players are playing a dice game.
+//Each one rolls a dice, the players who get the same number move to the next level.
+//Print the message displaying the players who moved to next level and players did not move to next level.
+
+
+Random dice = new Random();
+int player1 = dice.Next(1, 7);
+int player2 = dice.Next(1, 7);
+int player3 = dice.Next(1, 7);
+int player4 = dice.Next(1, 7);
+int player5 = dice.Next(1, 7);
+
+Console.WriteLine($"Player1 Dice Roll Number : {player1}\nPlayer2 Dice Roll Number:{player2} " +
+    $"\nPlayer3 Dice Roll Number : {player3} \nPlayer4 Dice Roll Number: {player4} \nPlayer5 Dice Roll Number: {player5}");
+
+if (((player1 == player2) || (player1 == player3) || (player1 == player4) || (player1 == player5)) &&
```

Comment:
player1 == player2 is same as player2 == player1.

Reply:
In your modified code you are checking for player1 == player2 and player2 == player3 and player1 == player3. For the same logic, you are verifying it 3 times. There should be some simplified version, think about it.